src/core/components/ChatView.module.css

```diff
@@ -280,6 +280,9 @@
 }
 
 .inputRaised {
+  width: -webkit-fill-available; /* WebKit browsers (Chrome, Edge, Safari) */
+  width: -moz-available;
+  width: fill-available;
```

Comment:
This is interesting. Previously did we used to have a fixed width on this? Curious did this change because of the new chat view? I'd almost say keep the input a consistent width the entire way instead of supporting the change